src/cli/migrate/nodeVersion/getNode22TypesVersion.ts

```diff
@@ -0,0 +1,4 @@
+import { execSync } from 'child_process';
+
+export const getNode22TypesVersion = () =>
+  execSync("npm show @types/node@^22 version --json | jq '.[-1]'").toString();
```

Comment:
Maybe we could generalise a bit: 

```suggestion
export const getNodeTypesVersionForMajor = (major: number) =>
  execSync(`npm show @types/node@^${major} version --json | jq '.[-1]'`).toString();
```